Doc/library/asyncio-api-index.rst

```diff
@@ -132,18 +132,43 @@ High-level APIs to work with network IO.
     :widths: 50 50
     :class: full-width-table
 
+    * - ``await`` :func:`connect`
+      -  Establish a TCP connection to send and receive data.
+
     * - ``await`` :func:`open_connection`
       -  Establish a TCP connection.
 
+    * - ``await`` :func:`connect_unix`
+      -  Establish a Unix socket connection to send and receive data.
+
     * - ``await`` :func:`open_unix_connection`
       -  Establish a Unix socket connection.
```

Comment:
You can add "(deprecated in favor of the XXX function)" for old APIs.

Reply:
Thanks, done.